<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fields</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="fieldsView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonLayout">
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="toolTip">
          <string>Add new field</string>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteButton">
         <property name="toolTip">
          <string>Delete selected field</string>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="upButton">
         <property name="toolTip">
          <string>Move selected field up</string>
         </property>
         <property name="text">
          <string>up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="downButton">
         <property name="toolTip">
          <string>Move selected field down</string>
         </property>
         <property name="text">
          <string>down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="resetButton">
         <property name="toolTip">
          <string>Reset all fields</string>
         </property>
         <property name="text">
          <string>reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loadFromLayerLayout">
     <item>
      <widget class="QLabel" name="loadFromLayerLabel">
       <property name="text">
        <string>Load fields from layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="layerCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadLayerFieldsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Load fields from selected layer</string>
       </property>
       <property name="text">
        <string>Load fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
